src/components/ImageView/CatalogView/CatalogViewGLComponent.tsx

```diff
@@ -37,12 +38,13 @@ export class CatalogViewGLComponent extends React.Component<CatalogViewGLCompone
         // dummy values to trigger React's componentDidUpdate()
         /* eslint-disable @typescript-eslint/no-unused-vars */
         const appStore = AppStore.Instance;
-        const baseFrame = appStore.activeFrame;
+        const baseFrame = this.props.frame;
```

Comment:
these changes are in preparation for #1259, because we will need to render frames other than the active one.